modules/income_and_assets/spec/lib/income_and_assets/notification_email_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'income_and_assets/notification_email'
+
+RSpec.describe IncomeAndAssets::NotificationEmail do
+  let(:saved_claim) { create(:income_and_assets_claim) }
+
+  describe '#deliver' do
+    it 'successfully sends an email' do
+      expect(IncomeAndAssets::SavedClaim).to receive(:find).with(23).and_return saved_claim
+      expect(Settings.vanotify.services).to receive(:income_and_assets).and_call_original
+
+      args = [
+        saved_claim.email,
+        Settings.vanotify.services['21p_0969'].email.submitted.template_id,
+        anything,
+        Settings.vanotify.services['21p_0969'].api_key,
```

Comment:
Does this not work? Most settings in the repo are accessed with dot notation.

```suggestion
        Settings.vanotify.services.21p_0969.api_key,
```

Reply:
Yeah, have it starting with a digit causes a syntax error
![image](https://github.com/user-attachments/assets/68b51121-1c92-4ebb-9013-ab1f1fba1cb4)

Reply:
ok, good to know. thanks!